milatools/cli/remote.py

```diff
@@ -467,7 +466,16 @@ def __init__(
         )
 
     def srun_transform(self, cmd: str) -> str:
-        return shlex.join(["srun", *self.alloc, "bash", "-c", cmd])
+        cmd = shlex.join(["srun", *self.alloc, "bash", "-c", cmd])
+        # NOTE: We need to cd to $SCRATCH before we run `sbatch` on DRAC clusters.
+        # self.connection.host can be something like `cedar.alliancecan.ca`
+        connection_host = self.connection.host
+        assert isinstance(connection_host, str)
+        if any(
+            connection_host.startswith(cluster_name) for cluster_name in DRAC_CLUSTERS
+        ):
+            cmd = f"cd $SCRATCH && {cmd}"
+        return cmd
```

Comment:
Would it be simpler to always do cd `$SCRATCH && {cmd}`?

Reply:
Not a blocker tho, this could be changed later

Reply:
Yeah, can't believe I hadn't thought of that before!

Reply:
Fixed this in 119c4214d748fa687f19375a5fc75d681141abb6